symengine/tests/test_functions.py

```diff
@@ -1,5 +1,5 @@
 from symengine import Symbol, sin, cos, sqrt, Add, Mul, function_symbol, Integer, log, E, symbols
-from symengine.lib.symengine_wrapper import Subs, Derivative
+from symengine.lib.symengine_wrapper import Subs, Derivative, _max, _min
```

Comment:
We don't want `_max` to be the public interface.
`Max` can be the public interface, but for that we need to rename the existing class to `_Max` and make `Max`, a pure python class and override `__new__` to return the value, like in https://github.com/symengine/symengine.py/blob/master/symengine/sympy_compat.py#L53-L54